packages/cactus-cmd-api-server/src/test/typescript/benchmark/artillery-api-benchmark.test.ts

```diff
@@ -0,0 +1,143 @@
+import test, { Test } from "tape-promise/tape";
+import { v4 as uuidv4 } from "uuid";
+import { JWK } from "jose";
+
+import { LogLevelDesc, LoggerProvider } from "@hyperledger/cactus-common";
+
+import {
+  PluginImportType,
+  ConsortiumDatabase,
+} from "@hyperledger/cactus-core-api";
+
+import {
+  ApiServer,
+  AuthorizationProtocol,
+  ConfigService,
+} from "../../../main/typescript/public-api";
+
+import { DefaultApi as ApiServerApi } from "../../../main/typescript/public-api";
+
+const log = LoggerProvider.getOrCreate({
+  level: "INFO",
+  label: "logger-test",
+});
+
+const util = require("util");
```

Comment:
@travis-payne the `import { ... } from "..."` syntax is preferred unless there's some specific reason to not use it. Sorry for the nitpicking!